CITATION.cff

```diff
@@ -0,0 +1,71 @@
+cff-version: 1.2.0
+message: "Please cite the following when using this software: https://doi.org/10.1117/12.2629569"
```

Comment:
I think the commonly accepted paradigm is to say something like "If you use this software, please cite it using this metadata." so that you don't have to update the link twice (here in the `message` and later in the `preferred-citation`).

Reply:
Yes, although I was worried that might lead to an attempt to cite the software itself which doesn't have a DOI or ASCL ID and so thought I should be explicit.